flaky-tests/get-started/ci-providers/bitbucket-pipelines.md

```diff
@@ -8,15 +8,15 @@ Before you start on these steps, see the [Test Frameworks](../frameworks/) docs
 
 ### 1. Store a TRUNK\_TOKEN secret in your CI system
 
-In [app.trunk.io](https://app.trunk.io/login/?intent=flaky+tests), navigate to:
+In [app.trunk.io](https://app.trunk.io/login?intent=flaky%20tests), navigate to:
 
 **Settings > Organization > Manage > Organization API Token > View Organization API Token > View**
 
 Store your API Token in your BitBucket  as a new variable named `TRUNK_TOKEN`. Make sure you are getting your _organization token_, not your project/repo token.
 
 ### 2. Grab your Organization Slug
 
-To upload test results to Trunk, you'll need to pass a Trunk Organization Slug to the upload command. To get your organization slug, In [app.trunk.io](https://app.trunk.io/login/?intent=flaky+tests), navigate to:
+To upload test results to Trunk, you'll need to pass a Trunk Organization Slug to the upload command. To get your organization slug, In [app.trunk.io](https://app.trunk.io/login?intent=flaky%20tests), navigate to:
```

Comment:
I'll fix up the extra comma in all of these pages separately
>To get your organization slug, In

Reply:
i'll leave it as-is and you can fix/update them (including this one) in another PR